feat(agentless): Add list of agent libraries (#387)
ngrok/ngrok-docs: docs/secure-tunnels/agentless.md

[Learn More](/docs/using-ngrok-with/docker)

## ngrok Agent Libraries

> ngrok Agent libraries allow you to embed ngrok functionality in your code without having to manage another process or bundling the ngrok binary in your project. Allowing you to control ngrok tunnels and configuration programmatically.

Reviewer: ```suggestion
Agent libraries embed ngrok in your app's code. You don't have to manage another process or bundle the ngrok agent with your project. They allow you to create ngrok endpoints and define configuration programatically.
```
